index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>John Buhendwa</title>
+</head>
+<body>
+    <header>
+        <div id="logo">Welcome</div>
```

Comment:
- [ ] You did an amazing job implementing the logo on your index.html :+1:, but in order to follow the correct [semantics of the HTML](https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md) you should wrap the logo with `<a>` tag instead of using `<div>`. This is for three main reasons, convention, semantics, and user experience(UX)